docs-md/app/materialized.md

```diff
@@ -0,0 +1,365 @@
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("michael", "help", 224);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "help", 802);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "purchase", 10204);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "help", 600);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "refund", 105);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("michael", "help", 2030);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("colin", "purchase", 800);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "help", 2514);
+INSERT INTO calls (name, reason, duration_seconds) VALUES ("derek", "refund", 325);
+```
+
+### Start the Debezium connector
+
+With MySQL ready to go, connect to ksqlDB's server using its interactive CLI. Run the following from your host:
+
+```
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+Before we issue more commands, instruct ksqlDB to start all queries from earliest point in each topic:
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Now we can connect ask Debezium to stream MySQL's changelog into Kafka. Invoke the following command in ksqlDB. This creates a Debezium source connector and writes all of its changes to Kafka topics:
+
+```sql
+CREATE SOURCE CONNECTOR calls_reader WITH (
+    'connector.class' = 'io.debezium.connector.mysql.MySqlConnector',
+    'database.hostname' = 'mysql',
+    'database.port' = '3306',
+    'database.user' = 'example-user',
+    'database.password' = 'example-pw',
+    'database.allowPublicKeyRetrieval' = 'true',
+    'database.server.id' = '184054',
+    'database.server.name' = 'call-center-db',
+    'database.whitelist' = 'call-center',
+    'database.history.kafka.bootstrap.servers' = 'broker:9092',
+    'database.history.kafka.topic' = 'call-center',
+    'table.whitelist' = 'call-center.calls',
+    'include.schema.changes' = 'false'
+);
+```
+
+After a few seconds, it should create a topic named `call-center-db.call-center.calls`. Print the raw topic contents to make sure it captured the initial rows that we seeded the calls table with:
+
+```sql
+PRINT 'call-center-db.call-center.calls' FROM BEGINNING;
+```
+
+If nothing prints out, the connector probably failed to launch. You can check ksqlDB's logs with:
+
+```
+docker logs -f ksqldb-server
+```
+
+You can also show the status of the connector in the ksqlDB CLI with:
+
+```
+DESCRIBE CONNECTOR calls_reader;
+```
+
+### Create the ksqlDB calls stream
+
+For ksqlDB to be able to use the topic that Debezium created, we need to declare a stream over it. Because we configured Kafka Connect with Schema Registry, we don't need to declare the schema of the data for the streams. It is simply inferred the schema that Debezium writes with. Run the following at the ksqlDB CLI:
```

Comment:
```suggestion
For ksqlDB to be able to use the topic that Debezium created, we need to declare a stream over it. Because we configured Kafka Connect with Schema Registry, we don't need to declare the schema of the data for the streams. It is simply inferred from the schema that Debezium writes with. Run the following at the ksqlDB CLI:
```